ui/src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -376,12 +376,19 @@ export const createOrUpdateTelegrafConfigAsync = () => async (
   )
 
   if (telegrafConfigID) {
-    const telegraf = await client.telegrafConfigs.update(telegrafConfigID, {
-      name: telegrafConfigName,
-      description: telegrafConfigDescription,
-      plugins,
+    const resp = await api.putTelegraf({
```

Comment:
Are you comparing this function from oats to the one you are replacing from the other client as you go and making sure they are entirely equivalent? It is making me really nervous that we are essentially rewriting the behavior of the app with these changes just to get label types to match. I would strongly prefer if we made these switches from one client to another each in a separate PR instead of one mega PR.

Reply:
Hey @ebb-tide , great question since the `putTelegraf` was the only one that's been a question in my mind with respect to this refactor. I did use the `generateRoutes` file in conjunction with the old apis to ensure that the endpoints, data, etc... were all the same:

https://raw.githubusercontent.com/influxdata/influxdb-client-js/master/src/api/api.ts

As I mentioned earlier, this `putTelegraf` one was (and still is) a question mark since we currently don't require the addition of `plugins` for `putTelegraf` as part of the parameters. I'd love to discuss this if you're around today